vert.x-web kotlin application fails on intellij 2019.3 ad win 10 1909

**Describe the bug**
downloaded a maven+kotlin skel. from code.quarkus.io. It is a REST config with RestEasy + vert.x-web. Running the code as is returns no problem. I've substituted the default "ExampleResource.kt" with a "hello world" from the vert.x-web kotlin guide:
```
package org.acme

import io.vertx.ext.web.Router

class Server : io.vertx.core.AbstractVerticle() {
    override fun start() {
        var server = vertx.createHttpServer()

        var router = Router.router(vertx)

        router.route().handler { routingContext ->

            // This handler will be called for every request
            var response = routingContext.response()
            response.putHeader("content-type", "text/plain")

            // Write to the response and end it
            response.end("Hello Worlds from Vert.x-Web!")
        }

        server.requestHandler(router).listen(8080)
    }
}
```
and the system blames me for targeting JVM 1.6, but JVM 1.8 is targeted in the pom.xml file

**Expected behavior**
It is expected to see the code run both in dev and in package and serve a static hello world (text/plain) page on the browser in localhost

**Actual behavior**
either in dev or in package run, maven fails with the following message:

>  Failed to execute goal org.jetbrains.kotlin:kotlin-maven-plugin:1.3.61:compile (compile) on project vert.x-skel: Compilation failure
[ERROR] C:\Users\XXX\IdeaProjects\vert.x-quarkus-skel\src\main\kotlin\org\acme\ExampleResource.kt:[9,29] Calls to static methods in Java interfaces are prohibited in JVM target 1.6. Recompile with '-jvm-target 1.8'

**To Reproduce**
Steps to reproduce the behavior:
1. download a MAVEN+Kotlin project where you add vert.x-web to the default RESTEasy
2. substitute the default "hello world" with the one in the vert.x-web kotlin docs
3. run the app under intellij 2019.03 community on Win 10 64 bit

**Configuration**
```properties
I attach my pom.xml, nothing added in the application.properties or similar
```
[this is my pom](https://gist.github.com/absinthetized/f27fa6a6b36e45b56280bc053657428d) (nothing chenged from the original, downloaded from code.quarkus.io)

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: 
 - Output of `java -version`: 
C:\Users\XXX\IdeaProjects\vert.x-quarkus-skel>java -version
java version "1.8.0_241"
Java(TM) SE Runtime Environment (build 1.8.0_241-b07)
Java HotSpot(TM) 64-Bit Server VM (build 25.241-b07, mixed mode)

 - GraalVM version (if different from Java): None used
 - Quarkus version or git rev: 1.3.0.Final
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: C:\Users\XXX\.m2\wrapper\dists\apache-maven-3.6.3-bin\1iopthnavndlasol9gbrbg6bf2\apache-maven-3.6.3
Java version: 1.8.0_202, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk1.8.0_202\jre
Default locale: it_IT, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"


**Additional context**
None I can think of...

--- a/devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/kotlin/pom.xml-template.ftl
+++ b/devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/kotlin/pom.xml-template.ftl
@@ -107,6 +107,7 @@
                 </executions>
                 <configuration>
                     <javaParameters>true</javaParameters>
+                    <jvmTarget>1.8</jvmTarget>
                     <!-- Soon to be replaced by plugin that will pre-configure all necessary annotations -->
                     <compilerPlugins>
                         <plugin>all-open</plugin>

